webapp/src/components/table/tableRow.tsx

```diff
@@ -1,38 +1,37 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
-import React, {useEffect, useRef, useState, useMemo} from 'react'
+import React, {useEffect, useRef, useState, useMemo, useCallback} from 'react'
 import {FormattedMessage} from 'react-intl'
 
 import {Card} from '../../blocks/card'
 import {Board, IPropertyTemplate} from '../../blocks/board'
-import {BoardView} from '../../blocks/boardView'
 import {Constants} from '../../constants'
 import mutator from '../../mutator'
 import Button from '../../widgets/buttons/button'
 import Editable from '../../widgets/editable'
 import {useSortable} from '../../hooks/sortable'
-import {useAppSelector} from '../../store/hooks'
-
-import {getCardContents} from '../../store/contents'
-
-import {getCardComments} from '../../store/comments'
 
 import PropertyValueElement from '../propertyValueElement'
 import './tableRow.scss'
 
 type Props = {
     board: Board
-    activeView: BoardView
+    columnWidths: Record<string, number>
```

Comment:
Being more granular about the data passed to avoid rerenders when something outside of this list (but in the activeView) changes.